config/epochConfig.go

```diff
@@ -91,6 +91,7 @@ type EnableEpochs struct {
 	RefactorPeersMiniBlocksEnableEpoch                uint32
 	MaxBlockchainHookCountersEnableEpoch              uint32
 	WipeSingleNFTLiquidityDecreaseEnableEpoch         uint32
+	DynamicGasCostForDataTrieStorageLoadEnableEpoch   uint32
```

Comment:
also adapt tests in the tomlConfig_test.go ?

Reply:
Done.